File: 13_14/Java/Assignment 4/Banner.java
package business;

 public class Banner{

	 private int width;
	 private int height ;

	 public Banner() {
		
	 width = 10;
	 height = 100;
	 
	 }

	 public Banner(int w, int h) {
		 width = w;
		 height = h;
	 }

	 public int getWidth() {
		 return width;
	 }

	 public void setWidth(int w) {
		 width = w;
	 }

	 public int getHeight() {
		 return height;
	 }

	 public void setHeight(int h) {
		 height = h;
	 }

	 public double getArea(){
		 double area = width * height;
		 return area;
	 }




 }

File: Assignment3/GeoLib/Banner.cs
namespace GeoLib;
public class Banner
{
    private float width;
    private float height;
  
   public Banner()
   { 
     width = 20f;
     height = 15f;
   }
      
public Banner(float width ,float height)
{
   this.width = width;
   this.height = height;   
}
    public float Width
        {
            get { return width; }
            set { width = value; }
        }
    
     public float Height 
     {
        get { return height;}
        set { height = value;}
     }

     public virtual float Area()
     {
      return width * height;
     }

     
}
   